src/main/docs/guide/openApiViews/swaggerui/oauth2.adoc

```diff
@@ -82,7 +83,7 @@ public class OrderController {
 
 Do not forget to configure Micronaut Security accordingly:
 
-[configuration]
+[source,yaml]
```

Comment:
we want to use the configuration macro to output documentation not just in yaml

Reply:
fixed